src/lib.rs

```diff
@@ -74,7 +75,6 @@
     clippy::match_wild_err_arm,
     clippy::match_wildcard_for_single_variants,
     clippy::mem_forget,
-    clippy::mismatched_target_os,
```

Comment:
Unrelated changed?

Reply:
This lint is removed on current stable channel
```
warning: lint `clippy::mismatched_target_os` has been renamed to `unexpected_cfgs`
  --> src\lib.rs:78:5
   |
78 |     clippy::mismatched_target_os,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: use the new name: `unexpected_cfgs`
   |
   = note: `#[warn(renamed_and_removed_lints)]` on by default
```